swo_aws_extension/management/commands/order_process_aws_invitations.py

```
@@ -0,0 +1,22 @@
+from swo_aws_extension.management.commands_helpers import StyledPrintCommand
+
+config = Config()
+
+
+class Command(StyledPrintCommand):
+    """Check AWS invitation state."""
+
+    help = "Check AWS invitation states"
+    name = "order_process_aws_invitations"
+
+    def handle(self, *args, **options):  # noqa: WPS110
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)